src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -139,7 +127,7 @@ export const trackIntegrationEvent = (
     analytics_session_id: sessionId,
     organization_id: org?.id,
     role: org?.role,
-    integration_directory_active: isIntegrationDirectoryActive(org),
+    integration_directory_active: true,
```

Comment:
@NisanthanNanthakumar We can just get rid of the field `integration_directory_active`